frontends/web/src/routes/lightning/send/send.tsx

```diff
@@ -79,7 +77,7 @@ const InvoiceInput = ({ invoice }: InvoiceInputProps) => {
         </div>
       )}
     </>
-  );
+  ) : <Skeleton />);
```

Comment:
In general the Skeleton component should not replace the whole content, but only the parts that can't be rendered.

Ideally only Amount and FiatConversion are replaced with a Skeleton, but all the other div's etc. can already be shown.

Reply:
Done